cmd/lncli/cmd_payments.go

```diff
@@ -1028,6 +1028,10 @@ var queryRoutesCommand = cli.Command{
 			Usage: "(optional) the channel id of the channel " +
 				"that must be taken to the first hop",
 		},
+		cli.StringSliceFlag{
+			Name:  "ignore_pair",
+			Usage: "ignore directional node pair <node1>:<node2>",
```

Comment:
nit: add "can be specified multiple times"?

Reply:
good idea 👍 updated!